### DISCUSSION TITLE
can't add-namespace, namespace already exists

### DISCUSSION THREAD

**Discussion by ahmgam** [shellhub-io/shellhub#2750] [Q&A] [answered]
Hello there!

I've been trying to run v-0.12 for hours, it seems that I'm missing something

after getting all containers up, I logged in cli container, then wrote to create an account : 

`/cli add-user admin zzzzzz admin@example.com`

the account is created successfully, now it's namespace turn

`/cli add-namespace example admin`

I get the following  

```
Error: namespace already exists
Usage:
  cli add-namespace [flags]

Flags:
  -h, --help   help for add-namespace

ERRO[0000] namespace already exists 
```

I don't know where is the problem, mongo is accepting cli connection with no problems, how can I fix this issue?

**Comment by gustavosbarreto** [comment]
> after getting all containers up, I logged in cli container, then wrote to create an account :

You don't need to log into the container to create an account. Just run `./bin/cli` from the root project directory.

> I get the following

It looks like there's already a namespace with that name. Are you doing a fresh installation of ShellHub? Check MongoDB, as you probably already have a registered namespace with that name.

Run the following command and past results here:

```
./bin/docker-compose exec mongo mongo --eval "db.namespaces.find()" main
```

**Reply by ahmgam** [reply]
Thanks for quick response ! 

I run this command in mongo instance, I got this response 

```
MongoDB shell version v4.4.8
connecting to: mongodb://127.0.0.1:27017/main?compressors=disabled&gssapiServiceName=mongodb
Implicit session: session { "id" : UUID("c71d429b-0289-4c3a-81b4-158db886ecfc") }
MongoDB server version: 4.4.8
```

how can I make sure that this is the one I've created? 

note that I have the same error while creating any namespace with any name.

**Comment by gustavosbarreto** [comment, marked as answer]
@ahmgam  I've created a issue for this (#2753) so we can track it.
